internal/service/cloudbackupsnapshot/data_source_cloud_backup_snapshots.go

```diff
@@ -129,8 +129,8 @@ func dataSourcePluralRead(ctx context.Context, d *schema.ResourceData, meta any)
 	params := &admin.ListReplicaSetBackupsApiParams{
 		GroupId:      d.Get("project_id").(string),
 		ClusterName:  d.Get("cluster_name").(string),
-		PageNum:      conversion.Pointer(d.Get("page_num").(int)),
-		ItemsPerPage: conversion.Pointer(d.Get("items_per_page").(int)),
+		PageNum:      conversion.IntPtr(d.Get("page_num").(int)),
```

Comment:
naming wise `conversion.Pointer` and `conversion.IntPtr` look very similar but behave differently. Curious if we should rename our conversion function to something like `conversion.IntPtrIfNotZero` (this can be a follow up PR)